File: tests/practice_run_tests/python/solution_files/sol19.py
from __future__ import annotations 
from typing import Optional

class TreeNode:
  def __init__(self, val, left: Optional[TreeNode] = None, 
               right: Optional[TreeNode] = None):
    self.val = val
    self.left = left
    self.right = right

class Solution:
  def solve(self, root: Optional[TreeNode]) -> Optional[TreeNode]:
    if not root or not root.right:
      return None
    return root.right

File: tests/practice_run_tests/java/solution_files/Sol19.java
package tests.practice_run_tests.java.solution_files;

public class Sol19 {
  public static TreeNode solve(TreeNode root) {
    if (root == null) return null;
    return root.right;
  }
  public static class TreeNode {
    public int val;
    public TreeNode left;
    public TreeNode right;

    public TreeNode(int val, TreeNode left, TreeNode right) {
      this.val = val;
      this.left = left;
      this.right = right;
    }
  }
}
  